apps/test/unit/lib/levelbuilder/lesson-editor/activitiesEditorReduxTest.js

```diff
@@ -27,7 +27,7 @@ import _ from 'lodash';
 
 const getInitialState = () => ({
   levelKeyList: {},
-  activities: sampleActivities
+  activities: _.cloneDeep(sampleActivities)
```

Comment:
For my own understanding why do this need to be cloned?

Reply:
this was a tricky one. without doing the clone here, the sample activities were getting mutated which was causing other tests to break.